test/suite/run.py

```diff
@@ -119,6 +119,7 @@ def usage():
                                  be run without executing any.\n\
   -g      | --gdb                all subprocesses (like calls to wt) use gdb\n\
   -h      | --help               show this message\n\
+          | --hook name[=arg]    set up hooks from hook_name.py, with optional arg\n\
```

Comment:
I don't see `hook_name.py`. Oh, I get it now. Would it be clearer as `hook_<name>.py`?

Reply:
Fixed.